### PR TITLE
Add more actionable information to stats

### LINKED ISSUE #81: Citations don't have useful information
Many of the citations that I've seen don't have any useful associated information with them. For example, 
here is an example of a [California that cites the code of civil procedure](https://github.com/SuffolkLITLab/FormFyxer/files/10397599/0ae4847f7b3a552e5f75d5d412bfc5ef.pdf) (in the bottom right corner).  The relevant text is `Code of Civil Procedure, § 706.124`. Because that specific regex string isn't in [reporters_db](https://github.com/freelawproject/reporters-db), eyecite only captures that the `§` character is a chapter of some sort, but only returns `UnknownCitation('§', metadata=CitationBase.Metadata(parenthetical=None))`, which by default doesn't give any of the relevant information surrounding the citation. That sort of info is important to know if we want to suggest people remove citations from forms.

My best suggestion for moving forward is to use the `index` attribute of the object to get the position in the original text, and grab at least 10 tokens before and after the symbol for context, which we can print when we print the citation. The difficulty is recreating the tokenization process (I think they include whitespace as separate tokens).

### PR DESCRIPTION
**Pull request by nonprofittechy**
Fix #81 for now

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [build-system]
-requires = ["setuptools>=40.0.0", "pip", "spacy"]
+requires = ["setuptools>=40.0.0,<66", "pip", "spacy"]
```
Comment (BryceStevenWilley) [inline pyproject.toml:2]: Does setuptools 66 break anything?

File: formfyxer/lit_explorer.py [hunk 2]
```diff
@@ -794,6 +798,66 @@ def needs_calculations(text: Union[str, Doc]) -> bool:
     return False
 
 
+def highlight_text(sentence:str, fragments:str, highlight_class:str="highlight")->str:
+    highlighted = sentence
+    for frag in fragments:
+        highlighted = re.sub(frag, f'<span class="{ highlight_class }">{frag}</span>', sentence)
+    return highlighted
```
Comment (BryceStevenWilley) [inline formfyxer/lit_explorer.py:805]: This belongs in PDFStats, not in here. Should factor it out. There are a lot of different ways we might report back this data, and PDFStats is the HTML package.

**Review by BryceStevenWilley — CHANGES_REQUESTED** [review]
highlighting text should be in PDFStats, but other than that, LGTM.

File: pyproject.toml (hunk 1 above)
Comment (nonprofittechy) [reply pyproject.toml:2]: Breaks a pip install -e on Ubuntu

File: formfyxer/lit_explorer.py (hunk 2 above)
Comment (nonprofittechy) [reply formfyxer/lit_explorer.py:805]: Fair enough. Was having trouble with passing a better data structure and serializing it, but I can do just a dictionary or something similar.

**Pull request merged**